src/client/driver/driver.js

```diff
@@ -1289,12 +1292,37 @@ export default class Driver extends serviceUtils.EventEmitter {
             });
     }
 
-    _onSetBreakpointCommand (isTestError) {
-        this.statusBar.showDebuggingStatus(isTestError)
-            .then(stopAfterNextAction => this._onReady(new DriverStatus({
+    _onSetBreakpointCommand ({ isTestError, hasCompilerService }) {
+        const showDebuggingStatusPromise = this.statusBar.showDebuggingStatus(isTestError);
+
+        if (hasCompilerService) {
```

Comment:
We use the `compiler service mode` term.
So, it better to rename as `inCompilerService` here and below.